etc/kayobe/environments/ci-builder/stackhpc-ci.yml

```diff
@@ -89,3 +89,6 @@ stackhpc_docker_registry_password: !vault |
 # Username and password of the overcloud host image repository.
 stackhpc_image_repository_username: "{{ stackhpc_docker_registry_username }}"
 stackhpc_image_repository_password: "{{ stackhpc_docker_registry_password }}"
+
+# SELinux set to disabled mode.
+selinux_state: disabled
```

Comment:
This needs to go in `etc/kayobe/environments/ci-builder/kolla/globals.yml`. I'm adding it in https://github.com/stackhpc/stackhpc-kayobe-config/pull/626 so you could remove if you like.

Reply:
Done